### PR TITLE
New Port: Sunset Shaped from matty

### PR DESCRIPTION
**Pull request by Cebion**
# New Port for Sunset Shaped

## Game Information
- **Title**: Sunset Shaped
- **URL**: https://bucketfish.itch.io/sunset-shaped

### CFW Tests
Ensure your game has been tested on all major CFWs:
- [X] AmberELEC
- [X] ArkOS
- [X] JELOS

### Resolution Tests
Test all major resolutions:
- [X] 480x320
- [X] 640x480
- [X] 720x720 (RGB30)
- [X] Higher resolutions (e.g., 1280x720)

### CODE REVIEW

File: ports/sunsetshaped/port.json [hunk 1]
```diff
@@ -0,0 +1,24 @@
...
+      "mattyj513"
+    ],
+    "desc": "Sunset Shaped is a tight and responsive platformer with five challenging levels and a calming atmosphere.Featuring interesting and tough level designs, simple and accessible controls - a singular jump and movement keys, flexible movement to nail the most precise of jumps, sweet and peaceful soundtrack and sound design, infinite opportunities for challenge runs and modifications.",
+    "inst": "All game files are included.",
+    "genres": [
+      "arcade"
+    ],
+    "image": null,
+    "rtr": true,
+    "runtime": null,
```
Comment (kloptops) [inline ports/sunsetshaped/port.json:21]: No runtime specified in port.json.

**Review by kloptops — CHANGES_REQUESTED** [review]
👎

**Review by kloptops — DISMISSED** [review]
👍

File: ports/sunsetshaped/port.json
Comment (tabreturn) [inline ports/sunsetshaped/port.json:1]: Typo: missing space after stop at " atmosphere.Featuring "

**Review by kloptops — APPROVED** [review]
👍

**Pull request merged**